shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/screens/MainScreen.kt

```diff
@@ -80,7 +80,8 @@ fun MainScreen(
             }
             composable<SpeakersScreen> {
                 Speakers(
-                    onBack = rootNavController::popBackStack
+                    service.speakers.collectAsState().value,
```

Comment:
This value too could be coming from the ViewModel of the screen instead of from here